README.md

```diff
@@ -78,7 +80,7 @@ Once applied, please allow a few minutes to for DNS propagation.
 
   The most likely reason is because the [DNS01 challenge](https://cert-manager.io/docs/configuration/acme/dns01/) has yet to resolve. If using `SETUP_MANAGED_DNS`, you will need to update your DNS records to point to the Azure DNS zone nameserver.
 
-  Once the DNS record has been updated, you will need to delete all Cert Manager pods to retrigger the certificate request
+  Once the DNS record has been updated, you will need to delete all cert-manager pods to retrigger the certificate request
 
   ```shell
```

Comment:
Later in this file at `Testing Gitpod Workspaces`, We seem to mention that the Load Balancer URl will be printed? Is it true anymore? 🤔

Reply:
Good catch